### DISCUSSION TITLE
アーキテクチャ検討

### DISCUSSION THREAD

**Discussion by Y-Kanoh** [rodbb/lambic#170] [General]
# プロトタイプの作成

* Typescript を採用
* データの永続化は行わず、オンメモリで実装 → 後から永続化を考える
  * テストが書きやすくなる

## 作成方法
* 元の Lambic と別リポジトリを作成
* 製品名は Lambic のまま
  * ユーザへの認知度はあるため

### 宿題
* 新プロジェクト名称候補を上げること

**Comment by amderbar** [comment]
Brewery 醸造所
お酒(Lambic)を作る場所(リポジトリ)なのであればこう呼べばよいかと

**Reply by Y-Kanoh** [reply]
Lambicビールにも以下のような種類があるらしい。
[参考](https://ja.m.wikipedia.org/wiki/ランビック)

- pure(普通のLambic)
- Gueze
- Mars
- Faro
- Kriek

**Reply by amderbar** [reply]
モジュール名に良さそう
